maven-java/akkaserverless-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/pom.xml

```diff
@@ -19,7 +19,7 @@
     <jdk.target>11</jdk.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-    <akkaserverless-sdk.version>0.7.0-beta.10</akkaserverless-sdk.version>
+    <akkaserverless-sdk.version>@project.version@</akkaserverless-sdk.version>
```

Comment:
Actually we had already a solution for this, but it was not being used for the sdk version, but only for the maven-plugin version. 

We want sdk, testkit and maven-plugin to use `akkaserverless-sdk.version` and it's now set using `@project.version@`.